userspace/libsinsp/sinsp.cpp

```diff
@@ -671,7 +671,7 @@ void sinsp::open_gvisor(const std::string& config_path, const std::string& root_
 	set_get_procs_cpu_from_driver(false);
 }
 
-void sinsp::open_modern_bpf(unsigned long driver_buffer_bytes_dim, uint16_t cpus_for_each_buffer, bool online_only, const libsinsp::events::set<ppm_sc_code> &ppm_sc_of_interest, const libsinsp::events::set<ppm_tp_code> &tp_of_interest)
+void sinsp::open_modern_bpf(unsigned long driver_buffer_bytes_dim, uint16_t cpus_for_each_buffer, bool online_only, bool verbose, const libsinsp::events::set<ppm_sc_code> &ppm_sc_of_interest, const libsinsp::events::set<ppm_tp_code> &tp_of_interest)
```

Comment:
I'd probably hook this one to the sinsp log verbosity, instead of exposing the option in this method.

Reply:
For scap_open instead, it makes sense to keep the exposed option imho.

Reply:
great idea ty! yeah for scap right now we have to pass it explicitly :/